frontend/src/pages/VendorsByMarketPage.js

```diff
@@ -23,9 +26,39 @@ const VendorsByMarketPage = () => {
     fetchMarket(marketId);
   }, [marketId]);
 
+  const filteredVendorArray =
```

Comment:
Ha jól nézem ez itt duplikálva van, szóval célszerű lenne valami common helyre kiemelni.
Illetve maga a 'toLowerCase...'  rész is olyan mintha kiszervezhető lenne. 

Egyszerűsödhetne a kód ha valahogy egy függvényben átadom egyszer az objectet aminek az attributumain keresek illetve az attributumait is átadom egy listában. Így ilyen ciklikusan is lehetne keresgélni 🤔

Reply:
Ne adj isten még unit teszteket is lehetne rá írni, ha ki van szervezve egy pure functionbe.

Reply:
👍

Reply:
a ciklikus keresés nem került bele, de a kiszervezés az megvan